helix-rest/src/main/java/org/apache/helix/rest/common/datamodel/RestSnapShot.java

```diff
@@ -0,0 +1,13 @@
+package org.apache.helix.rest.common.datamodel;
+
+
+public class RestSnapShot {
```

Comment:
Have a kv map there at least? And simple kv set/get methods.

Reply:
Add a comment at class level that this Snapshot can extend Snapshot from common/core module if there is more generic snapshot.

Reply:
Thanks for the review. I plan to add the enum and other implementation for this class in further PR.

Reply:
Add a note for that. Will do the impl in next PR.